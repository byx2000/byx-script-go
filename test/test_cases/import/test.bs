import a
import b

println('main')
