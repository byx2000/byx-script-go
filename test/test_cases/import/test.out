d
c
b
a
main